### PR TITLE
Record active contributors

### PR DESCRIPTION
**Pull request by bhs**
Also, revise the specifics of the process to reflect the latest proposal per #488 .

Thanks again to @lukaszgryglicki for the help with this!

### CODE REVIEW

File: elections/2020/governance-committee-election.md [hunk 1]
```diff
@@ -164,15 +164,305 @@ election).
...
+* [youngbupark](https://github.com/youngbupark)
+* [yue9944882](https://github.com/yue9944882)
+* [yurishkuro](https://github.com/yurishkuro)
+* [yyyogev](https://github.com/yyyogev)
+* [z1c0](https://github.com/z1c0)
+* [zeitlinger](https://github.com/zeitlinger)
+* [zenmoto](https://github.com/zenmoto)
+* [ziqizh](https://github.com/ziqizh) (no email found)
+
+We will contact all of the above via a GitHub issue and via email (where possible) to register themselves to vote in the election via this Google form: [https://forms.gle/2VWUNA8oz9DP4C3SA](https://forms.gle/2VWUNA8oz9DP4C3SA)
```
Comment (SergeyKanzhelev) [inline elections/2020/governance-committee-election.md:467]: > "to register themselves to vote in the election"

may be read as actual election process. Perhaps expand a little bit saying that actual election will be made via CIVS.

Comment (bhs) [reply elections/2020/governance-committee-election.md]: I tried to clarify, thanks.

**Review by SergeyKanzhelev — APPROVED** [review]

**Comment by bhs** [comment]
I'll merge tomorrow unless @kateetak raises any concerns.

**Review by yurishkuro — APPROVED** [review]

**Comment by kateetak** [comment]
No concerns from me! All good!

**Pull request merged**